plugins/rtmp-services/data/services.json

```diff
@@ -2903,6 +2903,26 @@
                 "x264opts": "scenecut=0",
                 "output": "rtmp_output"
             }
+        },
+        {
+            "name": "Amazon IVS",
+            "protocol": "RTMPS",
+            "servers": [
+                {
+                    "name": "Global (RTMPS, recommended)",
+                    "url": "rtmps://ingest.global-contribute.live-video.net:443/app"
+                },
+                {
+                    "name": "Global (RTMP)",
+                    "url": "rtmp://ingest.global-contribute.live-video.net/app"
+                }
+            ],
```

Comment:
I'm not sure if this is correct. I do see that we have at least one service with mixed RTMP/RTMPS that doesn't specify a protocol. @tytan652 @derrod ?

Reply:
https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines

> *(Required if not RTMP/RTMPS)* The streaming protocol that the service use

Don't specify if it's RTMP and RTMPS, it's the only multi-protocol exception.

TLDR: `"protocol"` is to be removed.

Reply:
Fixed and force pushed.